con-base/src/test/java/ba/codecentric/base/validation/util/BeanValidatorTestUtils.java

```diff
@@ -0,0 +1,49 @@
+package ba.codecentric.base.validation.util;
+
+
+import org.hibernate.validator.internal.engine.ValidationContext;
+import org.hibernate.validator.internal.engine.constraintvalidation.ConstraintValidatorManager;
+import org.mockito.invocation.InvocationOnMock;
+import org.mockito.stubbing.Answer;
+import org.springframework.test.util.ReflectionTestUtils;
+
+import javax.validation.ConstraintValidator;
+import javax.validation.Validator;
+import java.lang.annotation.Annotation;
+import java.lang.reflect.ParameterizedType;
+import java.util.concurrent.ConcurrentHashMap;
+
+import static org.mockito.Matchers.any;
+import static org.mockito.Mockito.doAnswer;
+import static org.mockito.Mockito.spy;
+
+public class BeanValidatorTestUtils {
```

Comment:
When you copy something from internet at least try to refactor it. Currently whole class looks really bad and unreadable. I hope you know what this class is doing ;) Please refactor it.